<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="7" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20675" started-at="2017-03-13T02:30:06Z" finished-at="2017-03-13T02:30:27Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20675" started-at="2017-03-13T02:30:06Z" finished-at="2017-03-13T02:30:27Z">
      <class name="com.codelovers.timeset.frontend.HomePage">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="setUp" is-config="true" duration-ms="2544" started-at="2017-03-13T09:30:06Z" finished-at="2017-03-13T09:30:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="logIn()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logIn" is-config="true" duration-ms="4367" started-at="2017-03-13T09:30:09Z" finished-at="2017-03-13T09:30:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="PASS" signature="TC1_checkTitleHomePage()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="TC1_checkTitleHomePage" duration-ms="437" started-at="2017-03-13T09:30:13Z" finished-at="2017-03-13T09:30:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC1_checkTitleHomePage -->
        <test-method status="PASS" signature="logOut()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logOut" is-config="true" duration-ms="483" started-at="2017-03-13T09:30:14Z" finished-at="2017-03-13T09:30:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="PASS" signature="logIn()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logIn" is-config="true" duration-ms="792" started-at="2017-03-13T09:30:14Z" finished-at="2017-03-13T09:30:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="PASS" signature="TC2_checkTitleDailySettlement()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="TC2_checkTitleDailySettlement" duration-ms="853" started-at="2017-03-13T09:30:15Z" finished-at="2017-03-13T09:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC2_checkTitleDailySettlement -->
        <test-method status="PASS" signature="logOut()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logOut" is-config="true" duration-ms="140" started-at="2017-03-13T09:30:16Z" finished-at="2017-03-13T09:30:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="PASS" signature="logIn()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logIn" is-config="true" duration-ms="1609" started-at="2017-03-13T09:30:16Z" finished-at="2017-03-13T09:30:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="FAIL" signature="TC3_checkTitleBudgetRanking()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="TC3_checkTitleBudgetRanking" duration-ms="332" started-at="2017-03-13T09:30:18Z" finished-at="2017-03-13T09:30:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<予算達成ランキング> but was:<予算ランキング>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<予算達成ランキング> but was:<予算ランキング>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at com.codelovers.timeset.frontend.HomePage.TC3_checkTitleBudgetRanking(HomePage.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC3_checkTitleBudgetRanking -->
        <test-method status="PASS" signature="logOut()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logOut" is-config="true" duration-ms="157" started-at="2017-03-13T09:30:18Z" finished-at="2017-03-13T09:30:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="PASS" signature="logIn()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logIn" is-config="true" duration-ms="1053" started-at="2017-03-13T09:30:18Z" finished-at="2017-03-13T09:30:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="FAIL" signature="TC4_checkTitleBudgetSale()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="TC4_checkTitleBudgetSale" duration-ms="391" started-at="2017-03-13T09:30:19Z" finished-at="2017-03-13T09:30:20Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<昨年対比ランキング> but was:<Frontend>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<昨年対比ランキング> but was:<Frontend>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at com.codelovers.timeset.frontend.HomePage.TC4_checkTitleBudgetSale(HomePage.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC4_checkTitleBudgetSale -->
        <test-method status="PASS" signature="logOut()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logOut" is-config="true" duration-ms="149" started-at="2017-03-13T09:30:20Z" finished-at="2017-03-13T09:30:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="PASS" signature="logIn()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logIn" is-config="true" duration-ms="1072" started-at="2017-03-13T09:30:20Z" finished-at="2017-03-13T09:30:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="PASS" signature="TC5_checkTitlePosts()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="TC5_checkTitlePosts" duration-ms="266" started-at="2017-03-13T09:30:21Z" finished-at="2017-03-13T09:30:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC5_checkTitlePosts -->
        <test-method status="PASS" signature="logOut()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logOut" is-config="true" duration-ms="415" started-at="2017-03-13T09:30:21Z" finished-at="2017-03-13T09:30:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="PASS" signature="logIn()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logIn" is-config="true" duration-ms="826" started-at="2017-03-13T09:30:21Z" finished-at="2017-03-13T09:30:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="PASS" signature="TC6_checkTextAppear()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="TC6_checkTextAppear" duration-ms="65" started-at="2017-03-13T09:30:22Z" finished-at="2017-03-13T09:30:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC6_checkTextAppear -->
        <test-method status="PASS" signature="logOut()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="logOut" is-config="true" duration-ms="494" started-at="2017-03-13T09:30:22Z" finished-at="2017-03-13T09:30:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.codelovers.timeset.frontend.HomePage@5c3bd550]" name="tearDown" is-config="true" duration-ms="118" started-at="2017-03-13T09:30:23Z" finished-at="2017-03-13T09:30:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.codelovers.timeset.frontend.HomePage -->
      <class name="com.codelovers.timeset.frontend.LogInPage">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.codelovers.timeset.frontend.LogInPage@91161c7]" name="setUp" is-config="true" duration-ms="1901" started-at="2017-03-13T09:30:23Z" finished-at="2017-03-13T09:30:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="TC1_LogInWithBlankField()[pri:0, instance:com.codelovers.timeset.frontend.LogInPage@91161c7]" name="TC1_LogInWithBlankField" duration-ms="2178" started-at="2017-03-13T09:30:25Z" finished-at="2017-03-13T09:30:27Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ユーザー名かパスワードが無効です。] but found [×
ユーザー名かパスワードが無効です。]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ユーザー名かパスワードが無効です。] but found [×
ユーザー名かパスワードが無効です。]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.codelovers.timeset.frontend.LogInPage.TC1_LogInWithBlankField(LogInPage.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC1_LogInWithBlankField -->
      </class> <!-- com.codelovers.timeset.frontend.LogInPage -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
